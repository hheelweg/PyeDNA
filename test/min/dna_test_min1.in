dna_dye: Initial minimization solvent + ions
 &cntrl
  imin   = 1,			! Explicit solvent minimization
  maxcyc = 1000,		! # of minimization steps
  ncyc   = 500, 		! # of initial steepest descent minimization steps
  ntb    = 1,			! Volume periodic boundaries
  ntr    = 1,			! Position restraint on GROUP (below)
  iwrap  = 1,			! Wrap molecules back into periodic box when they diffuse out
  cut    = 8.0			! Cutoff radius (angstroms) beyond which interactions are ignored
/
Hold the DNA fixed
500.0
RES 1 40
END
END