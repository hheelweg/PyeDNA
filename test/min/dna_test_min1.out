
          -------------------------------------------------------
          Amber 22 SANDER                              2022
          -------------------------------------------------------

| Run on 01/30/2025 at 07:12:04

|   Executable path: /home/hheelweg/.conda/envs/AmberTools23/bin/sander
| Working directory: /net/clusterhn.cluster.pssclabs.com/home/hheelweg/Cy3Cy5/test/min
|          Hostname: n012

  [-O]verwriting output

File Assignments:
|  MDIN: dna_test_min1.in                                                      
| MDOUT: dna_test_min1.out                                                     
|INPCRD: dna_test.rst7                                                         
|  PARM: dna_test.prmtop                                                       
|RESTRT: dna_test_min1.ncrst                                                   
|  REFC: dna_test.rst7                                                         
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: mdcrd                                                                 
|MDINFO: mdinfo                                                                
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

dna_dye: Initial minimization solvent + ions                                   
 &cntrl                                                                        
  imin   = 1,			! Explicit solvent minimization                                
  maxcyc = 1000,		! # of minimization steps                                    
  ncyc   = 500, 		! # of initial steepest descent minimization steps           
  ntb    = 1,			! Volume periodic boundaries                                   
  ntr    = 1,			! Position restraint on GROUP (below)                          
  iwrap  = 1,			! Wrap molecules back into periodic box when they diffuse out  
  cut    = 8.0			! Cutoff radius (angstroms) beyond which interactions are ignored
/                                                                              
Hold the DNA fixed                                                             
500.0                                                                          
RES 1 40                                                                       
END                                                                            
END                                                                            

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
 getting new box info from bottom of inpcrd
|  INFO: Old style inpcrd file read

| peek_ewald_inpcrd: Box info found
|Largest sphere to fit in unit cell has radius =    36.698
| New format PARM file being parsed.
| Version =    1.000 Date = 01/29/25 Time = 11:09:13
 NATOM  =   56038 NTYPES =      17 NBONH =   55156 MBONA  =     946
 NTHETH =    1208 MTHETA =    1454 NPHIH =    2502 MPHIA  =    3018
 NHPARM =       0 NPARM  =       0 NNB   =   80482 NRES   =   18290
 NBONA  =     946 NTHETA =    1454 NPHIA =    3018 NUMBND =      54
 NUMANG =     100 NPTRA  =      64 NATYP =      36 NPHB   =       0
 IFBOX  =       1 NMXRS  =      75 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real             5433626
|     Hollerith         186406
|     Integer          1561586
|     Max Pairs       18679333
|     nblistReal        672456
|     nblist Int       2971599
|       Total           139105 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

     BOX TYPE: RECTILINEAR

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =      50, ntrx    =       1, ntwr    =       1
     iwrap   =       1, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       1, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   8.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =   0.00000

Energy minimization:
     maxcyc  =    1000, ncyc    =     500, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       0
     Box X =   78.672   Box Y =   73.396   Box Z =  117.138
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =   80       NFFT2 =   75       NFFT3 =  120
     Cutoff=    8.000   Tol   =0.100E-04
     Ewald Coefficient =  0.34864
     Interpolation order =    4

    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  defa
    ----- READING GROUP     1; TITLE:
 Hold the DNA fixed                                                          

     GROUP    1 HAS HARMONIC CONSTRAINTS   500.00000
 GRP    1 RES    1 TO    40
      Number of atoms in this group  =  1360
    ----- END OF GROUP READ -----

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:    18214

     Sum of charges from parm topology file =   0.00100416
     Forcing neutrality...

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

 ---------------------------------------------------
 APPROXIMATING switch and d/dx switch using CUBIC SPLINE INTERPOLATION
 using   5000.0 points per unit in tabled values
 TESTING RELATIVE ERROR over r ranging from 0.0 to cutoff
| CHECK switch(x): max rel err =   0.2738E-14   at   2.422500
| CHECK d/dx switch(x): max rel err =   0.8332E-11   at   2.782960
 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =   10003059
| TOTAL SIZE OF NONBOND LIST =   10003059


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1       6.7484E+06     4.1153E+05     8.6150E+07     H30       318

 BOND    =      181.8250  ANGLE   =     1646.8799  DIHED      =     1157.5658
 VDWAALS =  6867981.4538  EEL     =  -187108.8484  HBOND      =        0.0000
 1-4 VDW =    65028.0709  1-4 EEL =     -492.2874  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
     50      -1.8266E+05     5.5428E+00     9.4602E+02     H30       318

 BOND    =     9937.4567  ANGLE   =     1281.5922  DIHED      =     1146.2050
 VDWAALS =    18565.2790  EEL     =  -215160.6972  HBOND      =        0.0000
 1-4 VDW =      751.6929  1-4 EEL =     -526.6471  RESTRAINT  =     1341.4213
 EAMBER  =  -184005.1185


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    100      -1.8950E+05     2.4527E+00     4.1879E+02     H30       318

 BOND    =    10601.9373  ANGLE   =     1312.0784  DIHED      =     1145.8131
 VDWAALS =    18658.6240  EEL     =  -222930.1503  HBOND      =        0.0000
 1-4 VDW =      671.1730  1-4 EEL =     -526.4735  RESTRAINT  =     1570.9581
 EAMBER  =  -191066.9980


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    150      -1.9321E+05     3.0627E+00     5.7658E+02     H30       318

 BOND    =    10987.3456  ANGLE   =     1323.1373  DIHED      =     1145.7747
 VDWAALS =    19056.6642  EEL     =  -227540.8286  HBOND      =        0.0000
 1-4 VDW =      676.4325  1-4 EEL =     -526.3359  RESTRAINT  =     1666.3705
 EAMBER  =  -194877.8103


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200      -1.9605E+05     2.2702E+00     4.0545E+02     H30       318

 BOND    =    11254.4036  ANGLE   =     1310.4097  DIHED      =     1145.8341
 VDWAALS =    19590.6935  EEL     =  -231068.1583  HBOND      =        0.0000
 1-4 VDW =      675.9536  1-4 EEL =     -526.3392  RESTRAINT  =     1570.2380
 EAMBER  =  -197617.2030


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    250      -1.9806E+05     1.2783E+00     1.7893E+02     H30       318

 BOND    =    11453.6435  ANGLE   =     1301.5834  DIHED      =     1145.8807
 VDWAALS =    20141.5635  EEL     =  -233746.9872  HBOND      =        0.0000
 1-4 VDW =      675.7430  1-4 EEL =     -526.4384  RESTRAINT  =     1494.0646
 EAMBER  =  -199555.0116


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    300      -1.9965E+05     3.0491E+00     6.1189E+02     H30       318

 BOND    =    11628.4646  ANGLE   =     1291.2945  DIHED      =     1146.0262
 VDWAALS =    20809.0030  EEL     =  -236068.8472  HBOND      =        0.0000
 1-4 VDW =      675.7806  1-4 EEL =     -526.5843  RESTRAINT  =     1398.0750
 EAMBER  =  -201044.8627


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    350      -2.0098E+05     4.9419E+00     1.0298E+03     H30       318

 BOND    =    11795.8681  ANGLE   =     1288.9488  DIHED      =     1146.0087
 VDWAALS =    21415.7676  EEL     =  -238143.3685  HBOND      =        0.0000
 1-4 VDW =      675.6602  1-4 EEL =     -526.6151  RESTRAINT  =     1368.3485
 EAMBER  =  -202347.7301


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    400      -2.0227E+05     2.5299E+00     5.0402E+02     H30       318

 BOND    =    11966.8173  ANGLE   =     1291.6816  DIHED      =     1146.0039
 VDWAALS =    21827.5302  EEL     =  -240053.3664  HBOND      =        0.0000
 1-4 VDW =      675.9476  1-4 EEL =     -526.5463  RESTRAINT  =     1405.4215
 EAMBER  =  -203671.9320


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    450      -2.0330E+05     2.2503E+00     4.2757E+02     H30       318

 BOND    =    12149.2241  ANGLE   =     1311.4774  DIHED      =     1145.7932
 VDWAALS =    22139.9604  EEL     =  -241774.4107  HBOND      =        0.0000
 1-4 VDW =      674.8799  1-4 EEL =     -526.3024  RESTRAINT  =     1583.9581
 EAMBER  =  -204879.3781


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -2.0429E+05     2.0295E+00     3.9756E+02     H30       318

 BOND    =    12260.4074  ANGLE   =     1292.3043  DIHED      =     1145.9810
 VDWAALS =    22808.9376  EEL     =  -243364.4511  HBOND      =        0.0000
 1-4 VDW =      675.5049  1-4 EEL =     -526.5042  RESTRAINT  =     1414.2532
 EAMBER  =  -205707.8201


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    550      -2.1056E+05     3.9448E+00     5.4311E+02     NA       1381

 BOND    =    13783.4599  ANGLE   =     1285.2916  DIHED      =     1147.0447
 VDWAALS =    27820.2489  EEL     =  -256363.3220  HBOND      =        0.0000
 1-4 VDW =      685.0296  1-4 EEL =     -524.5744  RESTRAINT  =     1605.6191
 EAMBER  =  -212166.8217


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    600      -2.1271E+05     2.4231E+00     2.5319E+02     C27       284

 BOND    =    14162.1626  ANGLE   =     1275.7102  DIHED      =     1146.5977
 VDWAALS =    29660.8445  EEL     =  -260681.2780  HBOND      =        0.0000
 1-4 VDW =      622.8891  1-4 EEL =     -523.5434  RESTRAINT  =     1624.5969
 EAMBER  =  -214336.6173


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    650      -2.1449E+05     3.4014E+00     6.9029E+02     C4'       230

 BOND    =    14383.2254  ANGLE   =     1310.2325  DIHED      =     1146.0664
 VDWAALS =    31181.5701  EEL     =  -264169.9471  HBOND      =        0.0000
 1-4 VDW =      723.9509  1-4 EEL =     -525.2034  RESTRAINT  =     1456.7504
 EAMBER  =  -215950.1051


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    700      -2.1567E+05     1.7188E+00     1.9311E+02     C27       284

 BOND    =    14547.4837  ANGLE   =     1313.7327  DIHED      =     1145.7242
 VDWAALS =    31974.7752  EEL     =  -266317.8615  HBOND      =        0.0000
 1-4 VDW =      647.8317  1-4 EEL =     -526.6047  RESTRAINT  =     1549.0024
 EAMBER  =  -217214.9188


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    750      -2.1661E+05     1.4055E+00     1.5649E+02     P         286

 BOND    =    14690.9279  ANGLE   =     1312.0489  DIHED      =     1145.7093
 VDWAALS =    32734.6282  EEL     =  -268133.5841  HBOND      =        0.0000
 1-4 VDW =      657.1346  1-4 EEL =     -526.7184  RESTRAINT  =     1510.0882
 EAMBER  =  -218119.8537


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    800      -2.1740E+05     1.3229E+00     1.4237E+02     C4'       230

 BOND    =    14856.5369  ANGLE   =     1307.7716  DIHED      =     1145.7069
 VDWAALS =    33446.1358  EEL     =  -269794.0129  HBOND      =        0.0000
 1-4 VDW =      659.8285  1-4 EEL =     -526.3893  RESTRAINT  =     1502.3905
 EAMBER  =  -218904.4224


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    850      -2.1813E+05     1.3004E+00     1.5948E+02     H30       318

 BOND    =    15036.6073  ANGLE   =     1306.6039  DIHED      =     1145.8144
 VDWAALS =    34088.3114  EEL     =  -271358.7078  HBOND      =        0.0000
 1-4 VDW =      660.4628  1-4 EEL =     -526.2540  RESTRAINT  =     1521.6141
 EAMBER  =  -219647.1620


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    900      -2.1879E+05     1.1137E+00     1.1581E+02     C3'       249

 BOND    =    15164.5928  ANGLE   =     1304.6318  DIHED      =     1145.6699
 VDWAALS =    34752.9760  EEL     =  -272777.6492  HBOND      =        0.0000
 1-4 VDW =      667.1324  1-4 EEL =     -526.1498  RESTRAINT  =     1479.8679
 EAMBER  =  -220268.7961
